Misc/NEWS

```diff
@@ -11,7 +11,7 @@ Core and Builtins
 -----------------
 
 - Issue #29537: Restore runtime compatibility with bytecode files generated by
-  CPython 3.5.0 and 3.5.1, and adjust the eval loop to avoid the problems that
+  CPython 3.5.0/1/2, and adjust the eval loop to avoid the problems that
```

Comment:
Good point. I realised that "to 3.5.2" will describe the relevant range without reflowing the text, so I switched to that.